sample-apps/django_sample/README.md

```diff
@@ -29,7 +29,7 @@ Run `bin/dev` to run the app in development mode. This will...
 
 Open https://judoscale-django.requestcatcher.com in a browser. The sample app is configured to use this Request Catcher endpoint as a mock for the Judoscale Adapter API. This page will monitor all API requests sent from the adapter.
 
-Start the app via `bin/dev`, then open http://localhost:5000. Continue to reload this page to collect and report more request metrics. You will see Judoscale Adatper API requests logged in Request Catcher.
+Start the app via `bin/dev`, then open http://localhost:5100. Continue to reload this page to collect and report more request metrics. You will see Judoscale Adatper API requests logged in Request Catcher.
```

Comment:
`bin/dev` runs two processes (as [defined in `Procfile`](https://github.com/judoscale/judoscale-python/blob/7f24d65da376f0e1cf4f25a6a224fe5a5c73cd58/sample-apps/django_sample/Procfile)):

- `proxy` runs on port 5000. It proxies requests to port 5100.
- `web` runs the Django app on 5100.

The proxy adds the `HTTP_X_REQUEST_START` header so we can test the [request middleware](HTTP_X_REQUEST_START).

I think for you, the proxy is not running because you don't have Node installed (or at least `npx` is not available).

At any rate, I know the setup is a bit confusing, but I think port 5000 is correct here.

Reply:
I installed npm so now npx can run, but I get this warning `WARN exec The following package was not found and will be installed: judoscale-adapter-proxy-server`.
But I will change the port back to 5000.

Reply:
I think that message is expected the first time you run it. It should still work, though. Did you try hitting port 5000 after running it like this?

Reply:
Yes, works at 5000 now as well and I can see the request time, so I put it back.
![image](https://user-images.githubusercontent.com/5695572/179714509-280c4736-4680-4cbb-8c3b-9580f709d958.png)